pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/OsteoAboutYouChildV2.java

```diff
@@ -0,0 +1,102 @@
+import org.broadinstitute.ddp.util.ConfigUtil;
+import org.broadinstitute.ddp.util.GsonUtil;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Optional;
+
+@Slf4j
+public class OsteoAboutYouChildV2 implements CustomTask {
+
+    private static final String DATA_FILE = "patches/about-child-v2.conf";
+    private static final String STUDY_GUID = "CMI-OSTEO";
+    private static final String ACTIVITY_CODE = "ABOUTYOU";
+
+    private Config dataCfg;
+    private Gson gson;
+    private Config studyCfg;
+
+    private SqlHelper helper;
+
+    @Override
+    public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
+        File file = cfgPath.getParent().resolve(DATA_FILE).toFile();
+        if (!file.exists()) {
+            throw new DDPException("Data file is missing: " + file);
+        }
+
+        this.dataCfg = ConfigFactory.parseFile(file);
+
+        if (!studyCfg.getString("study.guid").equals(STUDY_GUID)) {
+            throw new DDPException("This task is only for the " + STUDY_GUID + " study!");
+        }
+        this.studyCfg = studyCfg;
+        this.gson = GsonUtil.standardGson();
+    }
+
+    @Override
+    public void run(Handle handle) {
+        helper = handle.attach(SqlHelper.class);
+        ActivityDao activityDao = handle.attach(ActivityDao.class);
+        SectionBlockDao sectionBlockDao = handle.attach(SectionBlockDao.class);
+        JdbiUmbrellaStudy jdbiUmbrellaStudy = handle.attach(JdbiUmbrellaStudy.class);
+        var studyDto = jdbiUmbrellaStudy.findByStudyGuid(studyCfg.getString("study.guid"));
+        long activityId = ActivityBuilder.findActivityId(handle, studyDto.getId(), ACTIVITY_CODE);
+        Optional<ActivityVersionDto> activeVersion = activityDao.getJdbiActivityVersion()
+                .getActiveVersion(activityId);
+        if (activeVersion.isEmpty()) {
+            throw new DDPException("Activity version not found");
+        }
+        long revisionId = activeVersion.get().getRevId();
+        long stableId = helper.getQuestionStableIdByCode(dataCfg.getString("searchSectionByQuestionCode"), studyDto.getId());
+        long blockId = helper.getBlockIdByStableId(stableId);
+        long sectionId = helper.getSectionIdByBlockId(blockId);
+        String guid = handle.attach(JdbiExpression.class).generateUniqueGuid();
+        long shownExpr = helper.insertExpression(guid, dataCfg.getString("shownExpr"));
+        helper.insertBlockExpression(blockId, shownExpr, revisionId);
+        FormBlockDef blockDefChild = gson.fromJson(ConfigUtil.toJson(dataCfg.getConfig("who_filling_q_child")), FormBlockDef.class);
+        sectionBlockDao.insertBlockForSection(
```

Comment:
After I removed your changes to ActivityService (too dangerous to try that on every single one of our 20 studies at this point!) and modified the .conf file (see my comments there) the page is looking like this:
<img width="579" alt="Screen Shot 2022-09-13 at 9 35 23 PM" src="https://user-images.githubusercontent.com/29984380/190040965-c79b882c-d8e4-4645-873c-6323fe69d66e.png">

Reply:
I think the reason for that **1.** is that you are inserting a new block to hold the question and the old block is still there.
I BELIEVE that if before calling `insertBlockForSection` you delete blocks in the section, that 1 will go away.
I am not really setup to try this stuff/did not have time to setup Osteo locally, but I would encourage you to try clean this last thing.